python/tvm/relay/frontend/tflite.py

```diff
@@ -247,6 +249,55 @@ def convert_softmax(self, op):
 
         return out
 
+    def convert_concatenation(self, op):
+        """ convert TFLite concatenation"""
+        try:
+            from tflite.Operator import Operator
+            from tflite.ConcatenationOptions import ConcatenationOptions
+            from tflite.BuiltinOptions import BuiltinOptions
+            from tflite.ActivationFunctionType import ActivationFunctionType
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
+        assert isinstance(op, Operator)
+        input_tensors = self.get_input_tensors(op)
+        output_tensors = self.get_output_tensors(op)
+        assert len(input_tensors) >= 1, "input tensors should greater than 1"
+        assert len(output_tensors) == 1, "output tensors should be 1"
+        in_exprs = [self.get_expr(input_tensor.tensor_idx) for input_tensor in input_tensors]
+        assert op.BuiltinOptionsType() == BuiltinOptions.ConcatenationOptions
+        op_options = op.BuiltinOptions()
+        concatenation_options = ConcatenationOptions()
+        concatenation_options.Init(op_options.Bytes, op_options.Pos)
+        concatenation_axis = concatenation_options.Axis()
+        fused_activation_fn = concatenation_options.FusedActivationFunction()
+        input_shape_length = len(input_tensors[0].tensor.ShapeAsNumpy())
+        axis_convert_map = {}
+        # TFLite is N H W C, our layout is N C H W
+        if input_shape_length == 1 or input_shape_length == 2:
+            # The rule is channel first (after N but before H, W).
+            # length of 1 means N*H*W*C, do nothing.
+            # length of 2 means N*H*W, C, do nothing.
+            pass
+        elif input_shape_length == 3:
+            # convert N C H*W to N H*W C
+            axis_convert_map = {1:2, 2:1}
```

Comment:
line 284 and line 287. One has space between `:`,  one haven't. Please unify the style.

Additionally, your way of axis_convert_map could avoid additional transpose symbol. Could you help to improve `convert_reshape` and `convert_squeeze` too?

Reply:
It's different between concat and reshape. To remove additional op, we have to  use code likes [code in tfcoreml](https://github.com/tf-coreml/tf-coreml/blob/1e9e90bad799db7eb8d5547bdbeb0cc1e5180007/tfcoreml/_shape_sensitive_layers.py#L282). I'll try to change those later with support models as much as possible.